job_templates/sag_tf/config_fed_client.conf

```diff
@@ -27,18 +27,21 @@ executors = [
         # difference VS received parameters and send the difference
         params_transfer_type = "DIFF"
 
-        # if training is true, the executor will expect
-        # the custom code need to send back the trained parameters
-        training = true
-
-        # if global_evaluation is true, the executor will expect the
-        # custom code to send back the evaluation metric
-        global_evaluation = true
+        # if train_with_evaluation is true, the executor will expect
+        # the custom code need to send back both the trained parameters and the evaluation metric
+        # otherwise only trained parameters are expected
+        train_with_evaluation = true
 
         # if launch_once is true, the executor will only call launcher.launch_task once
         # for the whole job, if launch_once is false, the executor will call launcher.launch_task
         # everytime it receives a task from server
         launch_once = true
+
+        # params converter from NVFlare side to 3rd party
+        from_nvflare_converter_id = "from_nvflare"
+
+        # params converter from 3rd party to NVFlare side
+        to_nvflare_converter_id = "to_nvflare"
```

Comment:
maybe add some further details here? regarding why it is needed "nvflare expects a flattened dict, while keras orgainzes model weights as .... "

Reply:
good suggestion, added more explanation!